docs/reks/introduction.rst

```diff
@@ -0,0 +1,246 @@
+.. _sec-reks:
+
+============================================
+Energy and gradient calculations with REKS
+============================================
+
+*******************
+Single-state REKS
+*******************
+
+In single-state REKS, only ground state is calculated and it can treat the
+state with multireference character. For example, a ethylene molecule
+with torsional angle of 90 degrees shows degenerate :math:`{\pi}` and
+:math:`{\pi}^*` orbitals, which are correspond to highest occupied molecular
+orbital (HOMO) and lowest unoccupied molecular orbital (LUMO), respectively.
+In this case the SCC-DFTB calculation shows oscillating phenomena for self-consistent
```

Comment:
I agree for direct calculation zero or low temperature, this will usually be unstable for cases like this. But restarting from high temperature and progressively reducing electronic temperature can find a solution with the same total energy and charges as the REKS solution, but with somewhat different electronic levels. The finite temperature results have degeneracies, but REKS does not:
Finite T=1K           REKS
-17.109  2.00000   -17.109  2.00000
-13.148  2.00000   -13.148  2.00000
 -9.448  2.00000    -9.448  2.00000
 -9.448  2.00000    -9.403  2.00000
 -9.306  2.00000    -9.279  2.00000
 -4.276  1.00012    -4.349  2.00000
 -4.276  0.99988    -6.597  0.00000
 10.466  0.00000     1.349  0.00000
 10.700  0.00000    10.700  0.00000
 10.700  0.00000    10.700  0.00000
 11.362  0.00000    11.351  0.00000
 17.473  0.00000    17.470  0.00000

Reply:
What is the molecule used in this result? Can you give me the geometry file?